src/Lykke.Job.BlockchainCashinDetector.AzureRepositories/DepositWalletKeyDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Lykke.Job.BlockchainCashinDetector.Core.Domain;
+
+namespace Lykke.Job.BlockchainCashinDetector.AzureRepositories
+{
+    public class DepositWalletKeyDto : IDepositWalletKey
```

Comment:
It looks like complex type, not a DTO, so make immutable, and interface is just unnecessary